### DISCUSSION TITLE
Does Orca Slicer pass build plate type to Klipper?

### DISCUSSION THREAD

**Discussion by greatguitartist** [OrcaSlicer/OrcaSlicer#4161] [Dev]
I currently use 3 different build surfaces on my RatRig V-Core 3.1 - A smooth PEI Sheet, a textured PEI sheet, and a BuildTak plate. I also use a Beacon Eddy Current Sensor that allows me to save different models of the build plate. Depending on which surface I put on the bed plate, the Z-offset positions are different. I want to write a macro that runs during print start to select a specific beacon build plate model based on the bed type that is defined in Orca Slicer. Is this information passed to Klipper, or is there a system variable I can use to determine what bed type I've chosen for my print?

**Comment by DoctorWiz** [comment]
Apparently not.  I was trying to do the same thing.  Orca/Prusa slicer has a parameter that can be passed called "plate_type" however it always passes a blank.  That was the only parameter I could find that sounded anything at all like the build plate type.  Could be a bug however that may get fixed in a future release.

**Comment by scarteo** [comment]
https://github.com/SoftFever/OrcaSlicer/wiki/bed-types
